test/unit-test/main.tf

```diff
@@ -20,6 +20,7 @@ module "prometheus" {
   enable_prometheus_affinity_and_tolerations = false
   enable_large_nodesgroup                    = false
   enable_thanos_compact                      = false
+  dependence_ingress_controller              = "ingress"
```

Comment:
This is confusing. Is this the name of the controller?

Reply:
Updated to test data